doc/getting_started.md

```diff
@@ -1,53 +1,208 @@
+  for this purpose but other similar adapters should work as well.
+
+
+### Python Dependencies
+
+This repository has a couple of Python dependencies. You can run
+```console
+$ pip install --user -r python_requirements.txt
+```
+to install those dependencies.
 
-* https://github.com/lowRISC/opentitan/pull/2587
-* https://github.com/lowRISC/opentitan/pull/2735
 
 ### Git Large File Storage (LFS)
 
-This project uses Git LFS for storing binaries like the SPI flash tool and the
-target FPGA bitstream on a remote server. The repository itself just contains
-diff-able text pointers to the binaries. It is recommended to install the
-`git-lfs` tool for transparently accessing the binaries. Alternatively, they
-can be downloaded manually from GitHub.
+This project uses Git LFS for storing binaries like a debian-compatible
+pre-built binary of the tool for loading the OpenTitan application binary over
+SPI and an example target FPGA bitstream on a remote server. The repository
+itself just contains diff-able text pointers to the binaries. It is recommended
+to install the `git-lfs` tool for transparently accessing the binaries.
+Alternatively, they can be downloaded manually from GitHub.
 
 You can run
-```
-	sudo apt install git-lfs
+```console
+$ sudo apt install git-lfs
 ```
 to install the `git-lfs` tool on your Ubuntu machine.
 
-### Downloading code via FTDI
+Alternatively, you can rebuild those binaries yourself from the
+[OpenTitan](https://github.com/lowRISC/OpenTitan) repository.
+
 
-Currently the target firmware is downloaded via a SPI FTDI interface using the
-[spiflash](https://docs.opentitan.org/sw/host/spiflash/README/) utility.
-A debian-compatible pre-built binary is available at
-`cw/cw305/bin/linux/spiflash`. See the following table for details on how to
-connect the FTDI interface to the CW305 FPGA target board.
+### Generating OpenTitan Binaries
 
-| FTDI Signal | CW305 FPGA | OpenTitan IO        |
-| ----------- | ---------- | ------------------- |
-| TCK         | JP3.A14    | TCK (IO_DPS0)       |
-| TDI         | JP3.A13    | TDI (IO_DPS1)       |
-| TDO         | JP3.A15    | TDO (IO_DPS2)       |
-| TMS         | JP3.B15    | TMS (IO_DPS3)       |
-| GPIOL0      | JP3.C12    | nTRST (IO_DPS4)     |
-| GPIOL1      | JP3.C11    | nSRST (IO_DPS5)     |
-| GPIOL2      | JP3.B14    | JTAG_SEL (IO_DPS6)  |
-| GPIOL3      | JP3.C14    | BOOTSTRAP (IO_DPS7) |
-| GND         | GND        | -                   |
+Instead of using the example binaries provided by this repository via Git LFS,
+you can regenerate them from the [OpenTitan](https://github.com/lowRISC/OpenTitan)
+repository.
 
-## Running capture and attack
+To this end, follow these steps:
 
-Sample capture run:
+1. Go to the root directory of the OpenTitan repository.
+1. Before generating the OpenTitan FPGA bitstream for the CW305 target board,
+   you first have to run
+```console
+$ ./hw/top_earlgrey/util/opentitan_earlgrey_reduce_flash_size.py
```

Comment:
TODO: Add details on how to update the AES IP parameters for testing.

Reply:
Note, this is no longer required. The corresponding parameters are now propagated up to the top level and driven by the CW305-specific FPGA top ;-)